cli/fathom_web/commands/serve.py

```diff
@@ -1,6 +1,7 @@
-import contextlib
+from contextlib import contextmanager
 from http.server import HTTPServer, SimpleHTTPRequestHandler
 import os
+from socketserver import ThreadingMixIn
```

Comment:
Why not just use:
```python
from http.server import ThreadingHTTPServer
```

I.e. the `http.server` module has a [`ThreadingHTTPServer` class](https://github.com/python/cpython/blob/91daa9d7224626dad4bb820924c01b3438ca6e3f/Lib/http/server.py#L143) already, though it sets `daemon_threads=True`, which is not the default.

I don't really grok what that attribute controls. My best guess:
* Any threads created are designated as daemon threads.
* Any running daemon threads will automatically be shut down when the server closes.

Would you know if that's right?

Reply:
No kidding! That looks new in 3.7. I didn't see it. Let's see if it works…

Reply:
Yep!

Reply:
Incidentally, I think there's a latent error in FathomFox wherein it sometimes prematurely closes tabs and then complains about invalid tab IDs, but I don't see how it could be related to this.

Reply:
Yeah I have encountered that before. I don''t think that is related either.